src/test/command.spec.ts

```diff
@@ -81,6 +82,51 @@ describe("Command", () => {
       const result = run();
       await expect(result).to.be.rejectedWith("foo");
     });
+
+    it("should resolve a numeric --project flag into a project id", async () => {
+      nock.disableNetConnect();
```

Comment:
If you *must* include these, you should but them in a before/after so that if the test fails it doesn't leave everything disabled (not that that should matter, but it may)